packages/components-js/tests/a11y/specs/others/carousel.a11y.ts

```diff
@@ -146,12 +154,29 @@ test('should expose correct initial accessibility tree', async ({ page }) => {
   expect(await getAttribute(slide3, 'aria-label')).toBe('3 of 3');
 });
 
-test.fixme('should expose correct initial accessibility tree when aria prop is defined', async ({ page }) => {
-  await initCarousel(page, { aria: "{'aria-label': 'Other Heading'}" });
+test('should expose correct aria-label on splide element if slotted heading is set', async ({ page }) => {
```

Comment:
Do we also need a test if `slottedHeading` is true and an `aria-label` is set manually?